packit/config/aliases.py

```diff
@@ -183,3 +210,38 @@ def get_koji_targets(*name: str, default=DEFAULT_VERSION) -> Set[str]:
 
 def get_all_koji_targets() -> List[str]:
     return run_command(["koji", "list-targets", "--quiet"], output=True).split()
+
+
+@functools.lru_cache(maxsize=None)
+@fallback_return_value(ALIASES)
+def get_aliases() -> Dict[str, List[str]]:
+    """
+    Function to automatically determine fedora-all, fedora-stable, fedora-development and epel-all
+    aliases.
+    Current data are fetched via bodhi client, with default base url
+    `https://bodhi.fedoraproject.org/'.
+
+    :return: dictionary containing aliases
+    """
+
+    bodhi_client = BodhiClient()
+    releases = bodhi_client.get_releases(exclude_archived=True)
+    aliases = defaultdict(list)
+    for release in releases.releases:
+
+        if release.id_prefix == "FEDORA" and release.name != "ELN":
+            name = release.long_name.lower().replace(" ", "-")
+            aliases["fedora-all"].append(name)
+            if release.state == "current":
+                aliases["fedora-stable"].append(name)
+            elif release.state == "pending":
+                aliases["fedora-development"].append(name)
+
+        elif release.id_prefix == "FEDORA-EPEL":
+            name = release.name.lower()
+            aliases["epel-all"].append(name)
```

Comment:
```python
  print([r.name.lower() for r in releases.releases if r['id_prefix']=='FEDORA-EPEL'])                                                                                                                                                  
  ['el-6', 'epel-7', 'epel-8']
```

vs.

```python
  ALIASES["epel-all"]
  ["epel-6", "epel-7", "epel-8"],
```

Reply:
ok but that is returned by bodhi client.
Is it required to be exactly the same?

Reply:
I'm fine with whatever works. I just pointed out the difference to make sure you're aware of it.

Reply:
but maybe I missed your point - I updated ALIASES so it returns the same values as function.